framer/scripts/framerConfig.js

```diff
@@ -314,6 +314,7 @@ export const componentSettings = {
       'gutterBottom',
       'internalDeprecatedVariant',
       'paragraph',
+      'varaint',
```

Comment:
There's a typo but why do we want to ignore it? Seems pretty important for prototyping to me.

Reply:
The variants is now a union of enums + string, my understanding was that we are excluding those props.

Reply:
> my understanding was that we are excluding those props.

Why?

Reply:
> Why?

I believe we would need to fix the Framer generation script, not sure it's worth it:

https://github.com/mui-org/material-ui/blob/41e81f229e8f878287ea082b88b5808061a189ad/framer/scripts/framerConfig.js#L289